query_dbtropes.py

```diff
@@ -1,20 +1,29 @@
+from typing import List, Tuple, Any
+
 from rdflib import Graph
 from rdflib import URIRef
 
+from filenames import *
+
 
-def retrieve_tropes(series_name):
+def retrieve_tropes(series_name: str) -> List[Tuple[Any, str]]:
+    """
+    Retrieve series related tropes from local dbtropes series datastore.
+    :param series_name (str): Name of a series
+    :return List[Tuple[Any, str]]: List of tropes occurring in that series
+    TODO(bulka): what is in the tuple except for series name? Update docstring pls.
+    """
     g = Graph()
-    series_data_name = 'series_data_'+str(series_name).lower()+'.nt'
+    series_data_name = get_series_dbtropes_filename(series_name)
     g.parse(series_data_name, format="nt")
 
-    series_full_name = URIRef("http://dbtropes.org/resource/Series/" + series_name)
+    series_full_name = URIRef("http://dbtropes.org/resource/Series/" + series_name.replace(' ', ''))
```

Comment:
Got an error about wrong uri (contained whitespace). This line fixes that for e.g. silicon valley
http://dbtropes.org/resource/Series/SiliconValley